alacritty/src/main.rs

```diff
@@ -36,8 +36,11 @@ mod daemon;
 mod display;
 mod event;
 mod input;
+// TODO: Move into macos.
 #[cfg(target_os = "macos")]
 mod locale;
```

Comment:
I'm happy to do this now, or immediately (before 0.6) in a follow-up PR.

Reply:
My policy is to always allow style improvements in PRs. If we can make our code better by restructuring things a bit, that sounds good to me, no need to be overly pedantic about separation there.

So feel free to go ahead and move this into the macos module.